src/status_im/ui/components/text_input_with_label/view.cljs

```diff
@@ -39,16 +39,11 @@
   (animate-label text props)
   (reset! (:value* props) text))
 
-(defn text-input-handlers [{:keys [on-focus on-blur on-change-text on-change on-submit-editing
-                                   auto-expanding multiline max-height ref]} props]
+(defn text-input-handlers [{:keys [on-focus on-blur on-change-text on-change
+                                   on-submit-editing max-height ref]} props]
   {:ref               #(do
                          (reset! (:input-ref* props) %)
-                         (when ref (ref %)))
-   :on-change         #(do
-                         (when (and auto-expanding multiline)
-                           (reset! (:content-height* props) (min (.-height (.-contentSize (.-nativeEvent %)))
-                                                                 max-height)))
-                         (when on-change (on-change %)))
```

Comment:
So `on-change` is not used?

Reply:
`:on-change` is standard `TextInput` event handler, no need for special treatment here once we drop the height setting on each time it's triggered.

Reply:
:+1: